e2e/test.js

```diff
@@ -50,40 +50,42 @@ test('Game links are different', async t => {
 });
 
 test('Sample Game', async t=> {
```

Comment:
This mean that `test` is called as `t`?
Why? Doesn't it make it less readable? (especially on *e2e/game_page.js*)

Reply:
Yes it does, that's copy paste from the testcafe examples, will change!